timeline.go

```diff
@@ -257,6 +257,7 @@ func doPost(cCtx *cli.Context) error {
 		return errors.New("content is empty")
 	}
 
+        ev.Tags = nostr.Tags{ }
```

Comment:
```suggestion
	ev.Tags = nostr.Tags{ }
```

Reply:
Please forgive me my silly question, but is there anything I have to next?

I requested a change but I have almost zero knowledge about GitHub's workflow and I do not understand whether if I was expected to do something for the next.

Thanks.